grumpy-runtime-src/runtime/module.go

```diff
@@ -123,7 +123,8 @@ func importOne(f *Frame, name string) (*Object, *BaseException) {
 	o, raised := SysModules.GetItemString(f, name)
 	if raised == nil && o == nil {
 		if c = moduleRegistry[name]; c == nil {
-			raised = f.RaiseType(ImportErrorType, name)
+			msg := fmt.Sprintf("No module named %s", name)
```

Comment:
In CPython, there are single quotes around the module name.

Reply:
I copy-pasted from the CPython output on OSX. But maybe from 3.6 instead of 2.7.

We should talk about this...

Reply:
You got it right:
* $ __python2 -c "import bubba"__  # --> ImportError: No module named bubba
* $ __python3 -c "import bubba"__  # --> ModuleNotFoundError: No module named 'bubba'